source/core/src/main/com/csse3200/game/components/mainmenu/MainMenuDisplay.java

```diff
@@ -148,15 +156,20 @@ public void changed(ChangeEvent changeEvent, Actor actor) {
         }
         table.add(diffBtnsTable);
         table.row();
-        if (canLoad) {
-            table.add(shouldLoadBtn);
-            table.row();
+        if(canLoad) {
+            for (TextButton button : new TextButton[]{ loadButton,
+                    howToPlayBtn, achievementsBtn, settingsBtn, exitBtn}) {
+                table.add(button);
+                table.row();
+            }
         }
+        else{
+            for (TextButton button : new TextButton[]{
+                    howToPlayBtn, achievementsBtn, settingsBtn, exitBtn}) {
+                table.add(button);
+                table.row();
+            }
 
-        for (TextButton button : new TextButton[]{
-                howToPlayBtn, achievementsBtn, settingsBtn, exitBtn}) {
-            table.add(button);
-            table.row();
```

Comment:
why is this changed? You can just keep the original code but replace `shouldLoadBtn` with `loadButton`. Then you don't need two for loops

Reply:
changed it!